Core/src/org/sleuthkit/autopsy/communications/relationships/SummaryViewer.java

```diff
@@ -110,7 +111,16 @@ public void setSelectionInfo(SelectionInfo info) {
             Account[] accountArray = info.getAccounts().toArray(new Account[1]);
             Account account = accountArray[0];
             
-            accountLabel.setText(account.getTypeSpecificID());
+            if (account.getAccountType().getTypeName().contains("PHONE")) {
+                String countryCode = PhoneNumUtil.getCountryCode(account.getTypeSpecificID());
+                accountLabel.setText(PhoneNumUtil.convertToInternational(account.getTypeSpecificID()));
+                accountCountry.setText("Country: " + countryCode); //NON-NLS
```

Comment:
Text visable to the user should be a bundle message.

Reply:
Fixed